Generic_Version/Experiments.ipynb

```diff
@@ -385,7 +481,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
Are these just incidental changes?

Reply:
Looks like JetBrains IDEs add these pieces of metadata by convention. I reverted it and edited it with vscode instead which does not. I also ran `nb-clean` on the notebook to clear it from superfluous metadata that were added earlier.